core/src/main/scala/com/microsoft/azure/synapse/ml/logging/SynapseMLLogging.scala

```diff
@@ -63,20 +70,19 @@ trait SynapseMLLogging extends Logging {
     logBase("constructor")
   }
 
-  def logFit[T](f: => T): T = {
-    logVerb("fit", f)
+  def logFit[T](f: => T, columns: Int = -1): T = {
+    logVerb("fit", f, columns)
   }
 
-  def logTrain[T](f: => T): T = {
-    logVerb("train", f)
+  def logTrain[T](f: => T, columns: Int = -1): T = {
+    logVerb("train", f, columns)
   }
 
-  def logTransform[T](f: => T): T = {
-    logVerb("transform", f)
+  def logTransform[T](f: => T, columns: Int = -1): T = {
+    logVerb("transform", f, columns)
   }
```

Comment:
Still using this -1 arg which isnt scala idiomatic. Please remove the default arg from this function as it should be provided in all cases right?

Reply:
Did that. Please check.